service-front/module/Application/view/layout/layout.twig

```diff
@@ -89,6 +89,18 @@
         </div>
     </header>
 
+    <div class="emergency-banner emergency-banner--notable-death dont-print" role="banner" data-nosnippet>
```

Comment:
Alignment of html spacing here needs tidying

Reply:
+1 Yeah I didn't realise we didn't yet have linting in pre-commit hooks. One to make a story for. :)